### PR TITLE
Issue/79/create change password view

### PR DESCRIPTION
**Pull request by Bedpaw**
done, look at realapi method

### CODE REVIEW

File: src/components/organisms/form/change-password.vue [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+</template>
+
+<script>
+import AButtonSubmit from 'atoms/button/submit';
+import AInput from 'atoms/input';
+import { api } from 'api/index';
+import { mixins } from 'mixins/base';
+
+export default {
+  name: 'change-password.vue',
```
Comment (Drego31) [inline src/components/organisms/form/change-password.vue]: Wrong component name

File: src/components/organisms/form/change-password.vue [hunk 2]
```diff
@@ -0,0 +1,72 @@
+<template>
+  <div class="f-pb-1">
+    <div v-if="!formSend">
+      <a-input
+        :disabled="blockForm"
+        placeholder="Type your new password"
```
Comment (Drego31) [inline src/components/organisms/form/change-password.vue]: In Polish please :)

File: src/components/organisms/form/change-password.vue [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        v-model="password"
+        type="password"
+      />
+      <a-input
+        :disabled="blockForm"
+        placeholder="Repeat your new password"
```
Comment (Drego31) [inline src/components/organisms/form/change-password.vue]: In Polish please :)

File: src/components/organisms/form/change-password.vue (hunk 1 above)
Comment (Bedpaw) [reply src/components/organisms/form/change-password.vue]: fixed

File: src/components/organisms/form/change-password.vue (hunk 2 above)
Comment (Bedpaw) [reply src/components/organisms/form/change-password.vue]: translated

File: src/components/organisms/form/change-password.vue (hunk 3 above)
Comment (Bedpaw) [reply src/components/organisms/form/change-password.vue]: translated

**Pull request merged**